http-server/apache2-web-server/httpd.def

```diff
@@ -1,9 +1,57 @@
 Bootstrap: docker
-From: httpd:latest
+From: httpd:2.4.37
+
+%help
+Apache2 web server running in a Singularity Container.
+For full tutorial, visit: https://github.com/sylabs/examples/tree/master/http-server/apache2-web-server
+
+usage:
+
+  setup:
+  $ mkdir -p httpd/logs
```

Comment:
Change this to `$ mkdir -p ~/httpd/logs`

Reply:
Well, in this case, the `httpd/` directory is in the `~/httpd/`.
The `~/httpd/` directory is not necessary, but it makes it cleaner.